### ISSUE TITLE
Static Data

### ISSUE THREAD

**Issue by jomberna** [cnguy/kayn#3]
Hi, i'm having problem with the static data - getChampions 
when i start my node server, and try to call the API, it's return ok 

but if I refresh the page, it's return null, and in my debug console, they send me a message of cache.

this problem is with cache ? can you help me ?

**Comment by cnguy** [comment]
@jomberna 

Can you show me your initialization code (**remove the API key when you post your next screenshot or code snippets**) as well as the code of the specific route you're having issue with?

**Comment by lucasbordini** [comment]
```
var router = express.Router();
var KindredAPI = require('kindred-api')
var REGIONS = KindredAPI.REGIONS
var debug = true
var k = KindredAPI.QuickStart('API-KEY', REGIONS.NORTH_AMERICA, debug)

router.get('/', function(request, response, next) {
    const config = { options: { champListData: 'all' }, region: REGIONS.NORTH_AMERICA }

    k.Static.champions(config).then(data => response.send(data)).catch(error => response.status(400).send(error))
});

module.exports = router;
```

**Comment by cnguy** [comment]
@lucasbordini 

Strange, this works perfectly for me right now.

```javascript
// simple code, should work like yours
var express = require('express')
var app = express()

var KindredAPI = require('kindred-api')
var REGIONS = KindredAPI.REGIONS
var QUEUES = KindredAPI.QUEUE_TYPES
var debug = true
var k = KindredAPI.QuickStart('API_KEY', REGIONS.NORTH_AMERICA, debug)

app.get('/', function (req, res) {
    const config = { options: { champListData: 'all' }, region: REGIONS.NORTH_AMERICA }

    k.Static.champions(config)
     .then(data => {
        console.log('\ndata is being sent to client\n')
        res.send(data)
    })
     .catch(error => res.status(400).send(error))
})

app.listen(3000, function () {
    console.log('Example app listening on port 3000!')
})
```

<img width="712" alt="kindredss" src="https://cloud.githubusercontent.com/assets/17214097/25850534/916deaf4-3477-11e7-9d02-063ef67e6aac.png">

Modify your catch code so that you can show me the error:

```javascript
.catch(error => { // brackets
         console.error(error) // add this
         res.status(400).send(error)
 })
```

**Comment by lucasbordini** [comment]
console show this 

![image](https://cloud.githubusercontent.com/assets/27768689/25850875/727745fa-349a-11e7-8f43-221e1d3a3e57.png)

**Comment by cnguy** [comment]
@lucasbordini 

I noticed you were using 2.0.31, and was able to replicate the issue. Upgrade to 2.0.34 (many common bugs are fixed).

https://github.com/ChauTNguyen/kindred-api/blob/master/CHANGELOG.md (scroll to 2.0.32 section at the bottom).

The issue is fixed in 2.0.32 (but you should upgrade to 2.0.34)! :)

**Comment by lucasbordini** [comment]
fixed, thanks
